research-hub-web/src/app/components/guide-category/guide-category.component.ts

```diff
@@ -6,7 +6,16 @@ import { Location } from '@angular/common';
 import { AnalyticsService } from '@services/analytics.service';
 import { AppComponentService } from '@app/app.component.service';
 import { Content } from '@model/Content';
-import { ContentTypeId } from '@services/options.service';
+
+export enum ContentTypeId {
```

Comment:
- [ ] This `ContentTypeId` and a few of these other `enum`s are being duplicated all over the place, refactor please :v: